### ISSUE TITLE
Collapsible image thumbnails

### ISSUE THREAD

**Issue by williamkray** [ajbura/cinny#916]
### Describe the problem

Sometimes a user sends an image that you would prefer not to see, such as:

- inappropriate content for the room that hasn't been moderated yet
- an overly distracting animated image on loop
- when other cinny users prefer to hide thumbnails by default to conserve space in the conversation view, and only open thumbnails when necessary

### Describe the solution you'd like

I would like Cinny to support:

- a collapsible button next to image or video thumbnails that can be used to shrink the space normally taken by that thumbnail down to a single line of text (presumably the file type and name of the file, such as `Image (screenshot.jpg)` or `Video (my_recording.mp4)`
- an option in user settings to default all thumbnails to be collapsed

### Alternatives considered

there are currently no alternatives to hiding a thumbnail in cinny. alternatives would require using a different client that supports this feature. I do not know of any.

**Comment by williamkray** [comment]
this feature has _some_ overlap with https://github.com/cinnyapp/cinny/issues/679

**Comment by duarm** [comment]
this would go well with the "open media with default application" instead of embedded players

**Comment by duarm** [comment]
the caveat from my suggestion is that, for it to work with all video players/image viewers, the files would need to be saved to disk, which is a problem since we're saving private(encrypted) files, cinny needs to clean it up or at least warn the user. mpv can handle piping of images/videos directly to it tho.

**Comment by williamkray** [comment]
that sounds like a completely different request, and i would suggest opening a separate issue for that so they can be tracked independently.

**Comment by duarm** [comment]
It is, just bringing up my particular use-case for collapsible image thumbnails

**Comment by kfiven** [comment]
This is fixed by Low-bandwidth mode, where image don't load unless you click.

**Comment by williamkray** [comment]
> This is fixed by Low-bandwidth mode, where image don't load unless you click.

i disagree, as there is no simple way to re-hide the image once it has been clicked without switching to another room, and doing so re-hides _all images_ again; so if you are looking for a specific image you have to re-click them all, potentially causing you to re-expose the offending image which you then cannot hide again, creating a frustrating cycle. 

also, the requested behavior would save screen real-estate, whereas the current low-bandwidth implementation still has large blocks taken up by a non-image as demonstrated in this screenshot:

![image](https://github.com/cinnyapp/cinny/assets/8409433/5735cea1-9cec-4404-b35d-5e3f6358163f)
